presto-native-execution/presto_cpp/main/PrestoServerOperations.cpp

```diff
@@ -185,22 +189,42 @@ std::string PrestoServerOperations::veloxQueryConfigOperation(
   return unsupportedAction(op);
 }
 
-std::string PrestoServerOperations::debugOperation(
+std::string PrestoServerOperations::taskOperation(
     const ServerOperation& op,
     proxygen::HTTPMessage* message) {
+  if (taskManager_ == nullptr) {
+    return "Task Manager not found";
+  }
+  const auto taskMap = taskManager_->tasks();
   switch (op.action) {
-    case ServerOperation::Action::kTask: {
+    case ServerOperation::Action::kGetDetail: {
```

Comment:
We can combine the two operations into one
kTask:
id=, then look for all the tasks
count=0, then fetch all the tasks without limitation.

Reply:
Yeah merging them would be nice concise-wise. Though separating them can make it easier to get the meaning directly from the target/action level. Shall we keep it separated in this case? Thanks